spec/javascript/e2e/SupplementalApplicationPage.e2e.js

```diff
@@ -40,4 +41,55 @@ describe('SupplementalApplicationPage', () => {
 
     browser.close()
   }, DEFAULT_E2E_TIME_OUT)
+
+  test('should allow preference updates via the Confirmed Preferences section', async () => {
+    let browser = await puppeteer.launch({ headless: HEADLESS })
+    let page = await browser.newPage()
+
+    await sharedSteps.loginAsAgent(page)
+
+    // The application used here must include a claimed and editable
+    // preference as its first preference for this test to be able to pass.
```

Comment:
As we extend our tests, we can solidify the content of the test application much in the same way as we have solidified the content of the test listings, to provide reliability.